docs/examples/overview.md

```diff
@@ -0,0 +1,14 @@
+`django-components` makes it easy to share components between projects
+([See how to package components](../concepts/advanced/component_libraries.md)).
+
+Here you will find public examples of components and component libraries.
+
+If you have components that would be useful to others, open a pull request to add them to this collection.
+
+### Icons
+
+- [djc-heroicons](https://pypi.org/project/djc-heroicons/) - Icons from HeroIcons.com for django-components.
```

Comment:
My initial idea for the examples section was to be a place where we could put examplary component definitions (e.g. table, dialog, autocomplete, etc).

But in the documentation we've mentioned `django-htmx-components` and `djc-heroicons`. So instead of using that section solely for components, I decided to include also 3rd party libraries, grouped in some categories - similar to how you have the "awesome lists" on Github.